pyiron_atomistics/lammps/potential.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
 
-from __future__ import print_function
+from __future__ import print_function, annotations
```

Comment:
```suggestion
```

Both the `print` function and type annotations should be in all python versions we support.